echo_server.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+import socket
+import os
+import mimetypes
+
+def response_ok(t):
+    okay = u"HTTP/1.1 200 OK\r\n"
+    uritype = u"Content-Type: " + t[0] + "\r\n"
+    c_len = u"Content-length: " + str(len(t[1]))
+    crlf = u"\r\n\r\n"
```

Comment:
This is misnamed a bit.  CRLF is really just one repetition of the pattern: `"\r\n"`, not a pair of them.